### ISSUE TITLE
I would like a Single Record type chart

### ISSUE THREAD

**Issue by 12ian34** [lightdash/lightdash#12446]
### Description

Single Record chart type, [like Looker](https://cloud.google.com/looker/docs/single-record-options).

### What problem does this solve?

Good for presentations or reports where a single record is concerned. Much cleaner and nicer to show a single record in this format than in a long horizontal scrolling format.

**Comment by jpetey75** [comment]
This didn't pick up support in over six months, so we're tidying up the backlog. If you'd still like it, comment here and we'll reopen.

**Comment by jasonlanderson** [comment]
Yes please, unless there is a good workaround.  We're currently using a custom Vega chart we made but it's pretty fragile.
